scheduler/src/cook/components.clj

```diff
@@ -66,11 +64,25 @@
     (resolve var-sym)))
 
 (def raw-scheduler-routes
-  {:scheduler (fnk [mesos-datomic framework-id mesos-pending-jobs-atom settings]
+  {:retrieve-url-path-fn (fnk [[:settings agent-query-cache]]
+                           (log/info "Agent query cache will be created using:" agent-query-cache)
+                           (let [{:keys [threshold ttl]} agent-query-cache
+                                 cache (-> {}
+                                           (cache/fifo-cache-factory :threshold threshold)
```

Comment:
Isn't fifo cache the one with the concat bug? I know it's already in there but might be worth thinking about before we marry ourselves to a config setting.